packages/mdc-chips/chip-set/foundation.js

```diff
@@ -99,14 +118,8 @@ class MDCChipSetFoundation extends MDCFoundation {
    * @private
    */
   handleChipInteraction(evt) {
-    const chipFoundation = evt.detail.chip.foundation;
-    if (this.adapter_.hasClass(cssClasses.CHOICE) || this.adapter_.hasClass(cssClasses.FILTER)) {
-      if (chipFoundation.isSelected()) {
-        this.deselect(chipFoundation);
-      } else {
-        this.select(chipFoundation);
-      }
-    }
+    const {chipId} = evt.detail;
```

Comment:
Since the MDCChipInteractionEventType has changed, I think we need to update the typedef in MDCChipFoundation to reflect that it now consists of a chipId instead of a foundation.